### ISSUE TITLE
Deleting Besluit raises 500 error

### ISSUE THREAD

**Issue by SilviaAmAm** [open-zaak/open-zaak#1710]
### Product versie / Product version

1.13.0

### Omschrijf het probleem / Describe the bug

Trying to delete a Besluit from the Besluiten API returns a 500 error. In the logs, the stack trace is:

```
2024-07-10 10:28:23,967 ERROR vng_api_common.exception_handling  Exception c74ecf89-d8f7-47f7-b777-3fdba7d39d7f ocurred
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/drf_spectacular/drainage.py", line 207, in wrapped_method
    return method(self, request, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/notifications_api_common/viewsets.py", line 208, in destroy
    response = super().destroy(request, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/vng_api_common/audittrails/viewsets.py", line 164, in destroy
    response = super().destroy(request, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/rest_framework/mixins.py", line 91, in destroy
    self.perform_destroy(instance)
  File "/usr/local/lib/python3.10/contextlib.py", line 79, in inner
    return func(*args, **kwds)
  File "/app/src/./openzaak/components/besluiten/api/viewsets.py", line 143, in perform_destroy
    super().perform_destroy(instance)
  File "/app/src/./openzaak/components/zaken/api/mixins.py", line 76, in perform_destroy
    super().perform_destroy(instance)
  File "/usr/local/lib/python3.10/site-packages/rest_framework/mixins.py", line 95, in perform_destroy
    instance.delete()
  File "/usr/local/lib/python3.10/site-packages/django/db/models/base.py", line 1132, in delete
    return collector.delete()
  File "/usr/local/lib/python3.10/site-packages/django/db/models/deletion.py", line 512, in delete
    signals.post_delete.send(
  File "/usr/local/lib/python3.10/site-packages/django/dispatch/dispatcher.py", line 176, in send
    return [
  File "/usr/local/lib/python3.10/site-packages/django/dispatch/dispatcher.py", line 177, in <listcomp>
    (receiver, receiver(signal=self, sender=sender, **named))
  File "/app/src/./openzaak/components/zaken/signals.py", line 52, in sync_zaakbesluit
    ZaakBesluit.objects.delete_for(instance)
  File "/usr/local/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/app/src/./openzaak/components/zaken/query.py", line 99, in delete_for
    obj = self.get(zaak=besluit.zaak, besluit=besluit)
  File "/usr/local/lib/python3.10/site-packages/django/db/models/query.py", line 637, in get
    raise self.model.DoesNotExist(
openzaak.components.zaken.models.zaken.ZaakBesluit.DoesNotExist: ZaakBesluit matching query does not exist.
```

### Stappen om te reproduceren / Steps to reproduce

I used this docker compose file: https://github.com/maykinmedia/open-archiefbeheer/blob/12fb45921c988bbe0408191eddda38f5ca12e6ef/backend/docker-services/openzaak/docker-compose.yaml
With this fixture: 
[data.json](https://github.com/user-attachments/files/16159509/data.json)

Then, I made a `DELETE` call to `/besluiten/api/v1/besluiten/f5fe89b7-8d91-48b3-ba06-00d55de5c492`.

In the admin, I do see the zaak besluit between `ZAAK_3 en 596270410 - BESLUIT_00` :thinking: 

### Verwacht gedrag / Expected behavior

204 response

**Comment by annashamray** [comment]
I think I got why it happens. 
I managed to reproduce this error with the `data.json` fixture file. 
The signal to delete a related zaak-besluit fails because there is no zaak-besluit in the database. 
The same signal to create zaakbesluit has not worked because of this line:
https://github.com/open-zaak/open-zaak/blob/7e1cd2d9d4d00b3cc18370922daa3940f1996bd0/src/openzaak/components/zaken/signals.py#L34 

We intentionally exclude fixture loading for signals, sorry I misled you, when I said that a zaakbesluit would be created automatically. Its not the case with fixtures. 

I'd suggest to include `ZaakBesluit` objects in the fixture and try again
